src/test/java/com/icanj/sponsorship/IcanjSponsorshipTest.java

```diff
@@ -0,0 +1,44 @@
+package com.icanj.sponsorship;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class IcanjSponsorshipTest {
+    private WebDriver driver;
+
+    @Parameters({"user_name", "pass_word"})
+    @BeforeTest
+    public void launch(String userName, String password) {
+        // Create a new instance of the Chrome driver
+        System.setProperty("webdriver.chrome.driver", "/home/jason/Downloads/chromedriver");
+        driver = new ChromeDriver();
+        driver.get("http://my.icanj.org/");
+        driver.findElement(By.id("j_username")).sendKeys(userName);
+        driver.findElement(By.id("j_password")).sendKeys(password);
+        driver.findElement(By.xpath("//button[contains(text(),'Sign in')]")).click();
+        driver.manage().window().maximize();
```

Comment:
Since we're going to log into my.icanj multiple times, let's refactor this and move it into a base class. So that we don't have to repeat this code over and over again. Something like this

### BaseClass.java

```
package package com.icanj.base;

import org.testng.annotations.Test;

public class BaseClass {
        private WebDriver driver;

	public void login()
	{
		// Login related code
	}
}
```

### IcanjSponsorshipTest.java

```
package com.icanj.sponsorship;

import org.testng.annotations.Test;

public class BaseClass extends BaseClass {

        @Test
        public void clickSponsor() {
                // first log in and then run rest of the script.
		login()

	}
}
```

Reply:
Alright, will do. I will refactor and send another pull request.